rest/route/user.go

```diff
@@ -86,6 +86,61 @@ func (h *userSettingsGetHandler) Run(ctx context.Context) gimlet.Responder {
 	return gimlet.NewJSONResponse(apiSettings)
 }
 
+////////////////////////////////////////////////////////////////////////
+//
+// GET /rest/v2/users/{user_id}
+
+type getUserHandler struct {
+	userId string
+}
+
+func makeGetUserHandler() gimlet.RouteHandler {
+	return &getUserHandler{}
+}
+
+// Factory creates an instance of the handler.
+//
+//	@Summary		Get user
+//	@Description	Get information about the given user
+//	@Tags			users
+//	@Router			/users/{user_id} [get]
+//	@Security		Api-User || Api-Key
+//	@Param			user_id		path		string						true	"User ID"
+//	@Param			{object}	body		model.HostRequestOptions	true	"parameters"
+//
+//	@Success		200			{object}	model.APIDBUser				"list of users"
+func (h *getUserHandler) Factory() gimlet.RouteHandler { return h }
+func (h *getUserHandler) Parse(ctx context.Context, r *http.Request) error {
+	h.userId = gimlet.GetVars(r)["user_id"]
+	return nil
+}
+
+func (h *getUserHandler) Run(ctx context.Context) gimlet.Responder {
+	usr, err := user.FindOneById(h.userId)
+	if err != nil {
+		return gimlet.MakeJSONInternalErrorResponder(errors.Wrap(err, "finding user by ID"))
+	}
+	if usr == nil {
+		// Try getting using the display name
```

Comment:
Do we need to fall back to searching by display name? Seems more reliable to use the user ID or their email as unique identifiers.